sql/moz-fx-data-shared-prod/telemetry_derived/desktop_funnel_activation_v1/query.sql

```diff
@@ -0,0 +1,108 @@
+WITH distinct_countries AS (
+  -- Some country codes appear multiple times as some countries have multiple names.
+  -- Ensure that each code appears only once and go with name that appears first.
+  SELECT
+    code,
+    name
+  FROM
+    (
+      SELECT
+        row_number() OVER (PARTITION BY code ORDER BY name) AS rn,
+        code,
+        name
+      FROM
+        `moz-fx-data-derived-datasets`.static.country_names_v1 country_names
+    )
+  WHERE
+    rn = 1
+),
+pop AS (
+  SELECT DISTINCT
+    client_id,
+    normalized_country_code AS country_code,
+    normalized_channel AS channel,
+    application.build_id AS build_id,
+    normalized_os AS os,
+    mozfun.norm.truncate_version(normalized_os_version, "minor") AS os_version,
+    environment.settings.attribution.source AS attribution_source,
+    environment.partner.distribution_id AS distribution_id,
+    coalesce(environment.settings.attribution.ua, '') AS attribution_ua,
+    DATE(submission_timestamp) AS date
+  FROM
+    telemetry.new_profile
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+    AND payload.processes.parent.scalars.startup_profile_selection_reason = 'firstrun-created-default'
+),
+raw_info AS (
+  SELECT
+    client_id,
+    submission_date,
+    any_value(days_seen_bits) AS days_seen_bits
+  FROM
+    telemetry.clients_last_seen
+  WHERE
+    submission_date
+    BETWEEN @submission_date
+    AND DATE_ADD(@submission_date, INTERVAL 8 DAY)
+  GROUP BY
+    client_id,
+    submission_date
+),
+eight_days_later AS (
+  SELECT
+    a.*,
+    b.days_seen_bits
+  FROM
+    pop a
+  LEFT JOIN
+    raw_info b
+  ON
+    (a.client_id = b.client_id AND DATE_DIFF(b.submission_date, a.date, DAY) = 6)
```

Comment:
Related to comment https://github.com/mozilla/bigquery-etl/pull/1674#discussion_r561324111

Reply:
`raw_info` is still a windowed query over 8 days and I don't think there's any need for that to exist.

Based on my mental model of the intent here, I think this query should be a join between (using terminology from the [forward-looking windows docs](https://docs.telemetry.mozilla.org/cookbooks/clients_last_seen_bits.html#retention-forward-looking-windows):
- Deduped new profile pings from day 0
- clients_last_seen from day 6

The clients_last_seen row from day 6 contains all the data we need for understanding days 0 through 6.

It appears, though, that perhaps there's an extra offset here. So we receive a new profile ping on day 0, then we want to measure activity on days 1 through 7 (is that correct?). In which case this becomes a join between:
- Deduped new profile pings from day 0
- clients_last_seen from day 7

I still think that we should populate partitions based on the Airflow run date (for least operational surprise) and that we should provision a view (this can be under telemetry_derived to have it not be user-facing) to handle changing the date. In this case we have:

- `date` exposed in the view = `DATE_SUB(submission_date, interval 7 day)` (and the view does not expose submission_date)
- new_profile `WHERE submission_date = DATE_SUB(@submission_date, interval 7 day)`
- clients_last_seen `WHERE submission_date = @submission_date`
- `coalesce(udf.bitcount_lowest_7(days_seen_bits), 0) >= 5 AS activated`

It is entirely possible that I am missing nuance here, but I want to make sure that complexity in this query is necessary to minimize possibility of subtle bugs.

Reply:
I used the existing data and queries to check if there is an offset and the join should be on clients_last_seen from day 6 to reproduce the number in the existing queries.